e2e/tests/ui/notebook-ui.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('Notebook and Runtime UI Test', () => {
 
     const modal = new NewNotebookModal(page);
     await modal.waitForLoad();
-    await modal.name().type(pyNotebookName);
+    await modal.name().type(pyNotebookName.replace('.ipynb', ''));
```

Comment:
Fix the test for now, did not come up with a fancier approach because I don't think the test should be e2e. We have the same test that verified name should be unique in our unit test already. I believe we should delete this from e2e